cmd/list/machinepool/nodepool.go

```diff
@@ -21,16 +21,18 @@ func listNodePools(r *rosa.Runtime, clusterKey string, cluster *cmv1.Cluster) {
 	// Create the writer that will be used to print the tabulated results:
 	writer := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
 
-	fmt.Fprintf(writer, "ID\tAUTOSCALING\tREPLICAS\tINSTANCE TYPE\tAVAILABILITY ZONE\tSUBNET\tINSTANCE PREFIX\t\n")
+	fmt.Fprintf(writer, "ID\tAUTOSCALING\tDESIRED REPLICAS\tCURRENT REPLICAS\tINSTANCE TYPE\tAVAILABILITY ZONE\tSUBNET\tINSTANCE PREFIX\tMESSAGE\t\n")
 	for _, nodePool := range nodePools {
-		fmt.Fprintf(writer, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t\n",
+		fmt.Fprintf(writer, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t\n",
 			nodePool.ID(),
 			printNodePoolAutoscaling(nodePool.Autoscaling()),
 			printNodePoolReplicas(nodePool.Autoscaling(), nodePool.Replicas()),
+			printNodePoolCurrentReplicas(nodePool.Status()),
 			printNodePoolInstanceType(nodePool.AWSNodePool()),
 			nodePool.AvailabilityZone(),
 			nodePool.Subnet(),
 			printNodePoolName(nodePool.AWSNodePool()),
+			printNodePoolMessage(nodePool.Status()),
```

Comment:
Can we leave the `message` out of the table and output it only on `--output=[json|yaml]`? Otherwise it's a very long table.

Reply:
Note that when the https://github.com/openshift/rosa/pull/1019 is consolidated it will eliminate the "instance prefix" column and it will look pretty similar as the typical controller list in "kubectl"

Reply:
I think `message` is still important to show at the table level, even if it does increase table width